base/docs/basedocs.jl

```diff
@@ -1999,6 +1999,39 @@ for bit in (8, 16, 32, 64, 128)
     end
 end
 
+"""
+	$(Int) <: Signed
+
+`Int` is aliased to `Int64` on 64-bit systems and `Int32` on 32-bit systems.
```

Comment:
Can you actually access this docstring and `Int64`'s independently?

Reply:
Not that I know of. This shows up the docstring for Int64.

Reply:
So you get both of them or this shadows the other one?

Reply:
It shadows.